utils/endpoints/asset.js

```diff
@@ -62,6 +70,29 @@ export const getAsset = async (assetType, name) => {
           asset.size = size
           asset.highPoly = highPoly
           asset.file = CDN_URL(file.Key)
+          try {
+            const io = new NodeIO()
+              .registerExtensions([DracoMeshCompression, MaterialsUnlit])
+              .registerDependencies({
+                'draco3d.decoder': await draco3d.createDecoderModule(),
+                'draco3d.encoder': await draco3d.createEncoderModule(),
+              })
+            const json = await fetch(asset.file).then((rsp) => rsp.json())
+            const document = io.readJSON({ json })
```

Comment:
The current error shown sounds like the `readJSON` method requires both the .json and .resources properties of the [JSONDocument interface](https://gltf-transform.donmccurdy.com/interfaces/core.jsondocument.html), even though this glTF (using Data URIs) doesn't have any external resources. Something like this might fix it:

```js
const document = io.readJSON({ json, resources: {} });
```

Reply:
Actually never mind the comment above, this seems to be a bug in how I've implemented the Draco extension — it doesn't support decompressing Data URIs at the moment. Filed https://github.com/donmccurdy/glTF-Transform/issues/254.

Reply:
Will then subscribe to that issue and try again when it's fixed, no rush

Thank you so much for your help :)